static/webmention.js

```diff
@@ -230,7 +244,7 @@ A more detailed example:
                 linktext = extractComment(c);
             } else {
                 linkclass = "name";
-                linktext = "(mention)";
+                linktext = "(" + t("mention") + ")";
```

Comment:
For things like this I'd rather move to [template literals](https://caniuse.com/template-literals), e.g.

```
linktext = `(${t("mention")})`
```

Reply:
I was thinking about adjusting some other parts to „modern JavaScript™” as well, since terser should transpile them down anyway.
Can I scope this remark out here?

Reply:
Sure